app/views/spree/users/_saved_cards.html.haml

```diff
@@ -10,7 +10,7 @@
     %td.number{ ng: { bind: '::card.number' } }
     %td.expiry{ ng: { bind: '::card.expiry' } }
     %td.is-default
-      %input{ type: 'radio', name: 'default_card', ng: { model: 'card.is_default', change: 'setDefault(card)', value: "true"} }
+      %input{ type: 'radio', name: 'default_card', ng: { model: 'card.is_default', click: 'confirmSetDefault(card, $event)', value: "true"} }
```

Comment:
Not sure about using click events with radio buttons... it could technically be clicked without changing the selection, right?

Reply:
I have it as `click` because otherwise, if the user chooses not to confirm the change in the alert, the change has already happened and the UI is now incorrect (since we never send the response to the server). I looked around for a bit and didn't find a good solution that didn't involve storing the previous default card value and then re-selecting that radio button. 

What would really be nice is if we did something besides the native `confirm` javascript call... but I thought this was good enough for now.